t/memory.t

```diff
@@ -24,27 +24,31 @@ my $accounts =
 
 my %accounts = map { lc($_) => 1 } $accounts->@*;
 
+my $decoded_transaction = decode_json_utf8(
```

Comment:
just as a general comment, would be nice if we had a specific file for the data that we use so we wouldn't have the need to hardcode json structures like that in the test files.